src/main/resources/bidder-config/adtelligent.yaml

```diff
@@ -50,6 +50,15 @@ adapters:
             url:
             redirect-url:
             cookie-family-name: janet
+      pgam:
```

Comment:
Pretty sure that according to Go repo pgam has banner and video for app and site. Can you explain why you excluded them here?

Reply:
Aliases support the same configuration options that their bidder counterparts support except aliases (i.e. it’s not possible to declare alias of an alias). Create bidder alias [resource](https://docs.prebid.org/prebid-server/developers/add-new-bidder-java.html#plan-your-bid-adapter)